### PR TITLE
Linting using JavaScript Standard Style

### PR DESCRIPTION
**Pull request by nicolasnoble**
Using [JavaScript Standard Style](https://www.npmjs.com/package/standard)

The [VSCode extension](https://marketplace.visualstudio.com/items?itemName=shinnn.standard) can prove useful.

### CODE REVIEW

**Comment by nicolasnoble** [comment]
Better [VSCode extension](https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs).

File: src/games/tic_tac_toe/tic_tac_toe.js [hunk 1]
```diff
@@ -21,68 +21,68 @@ exports.setUp = (players) => {
...
+      let classList = []
+      let actions = []
       if (state.board[x][y].owner) {
-        class_list = [state.player_sides[state.board[x][y].owner]];
+        classList = [state.player_sides[state.board[x][y].owner]]
       } else {
-        class_list = ['empty']
-        if (!state.winner) actions = [
+        classList = ['empty']
+        actions = state.winner && [
```
Comment (murgatroid99) [inline src/games/tic_tac_toe/tic_tac_toe.js]: Where did this change come from?

**Comment by murgatroid99** [comment]
Most of these changes seem fine to me, but the spaces after function names are making me a little uncomfortable.

File: src/games/tic_tac_toe/tic_tac_toe.js (hunk 1 above)
Comment (nicolasnoble) [inline src/games/tic_tac_toe/tic_tac_toe.js]: The linter really wants a { after the if, and I didn't feel like rewriting the whole block :-)

Comment (nicolasnoble) [inline src/games/tic_tac_toe/tic_tac_toe.js]: Rewriting, because the indentation would've been off, of course.

**Comment by nicolasnoble** [comment]
Yeah, I agree with that one - and the switch indentation is ugly as well. It looks a lot of projects is using that style guide however, so maybe that'll help with readability and inclusiveness of other people's contributions ?

**Comment by murgatroid99** [comment]
I won't push back. But I do want to point out that even though it's called "standard", it's not actually a standard. It's just a code style they picked.

**Comment by nicolasnoble** [comment]
Yes, that much is clear, I agree :-) My only concern is having _a_ standard that a lot of people seems to have agreed on.

**Pull request merged**